synapse/lib/cell.py

```diff
@@ -709,6 +709,10 @@ async def _ctorNexsRoot(self):
     async def getNexsIndx(self):
         return await self.nexsroot.index()
 
+    @s_nexus.Pusher.onPushAuto('nexslog:setindex')
```

Comment:
I thought we didn't need to do this? Did we hit an edge case?

Reply:
This would **require** mirrors to be updated **before** the master is updated, correct?

Reply:
Yeah the mirror needs to be updated so it handles the event properly.